### PR TITLE
Micro Optimizations. Erase non-informative comments. Remove unused imports. Did TODO to load asset error image. Removed unused variables.

### PR DESCRIPTION
**Pull request by aa-rodriguezv**
## What does this PR accomplish?

Just minor changes that can help the app perform jsut a little bit better and reduce the apk size.

## Did you add any dependencies?

No

## How did you test the change?

I created my own firebase project and verified correct funcitoning of the app.

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -33,7 +33,7 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "com.thealphamerc.flutter_twitter_clone_dev"
```
Comment (TheAlphamerc) [inline android/app/build.gradle:36]: @aa-rodriguezv application Id cannot be changed, so please revert this to old one.

File: android/app/src/debug/AndroidManifest.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.thealphamerc.flutter_twitter_clone">
+    package="com.flutter_twitter_clone">
```
Comment (TheAlphamerc) [inline android/app/src/debug/AndroidManifest.xml:2]: @aa-rodriguezv package should not be changed, so please revert this to the old one.

File: android/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.thealphamerc.flutter_twitter_clone">
+    package="com.flutter_twitter_clone">
```
Comment (TheAlphamerc) [inline android/app/src/main/AndroidManifest.xml:2]: @aa-rodriguezv package should not be changed, so please revert this to the old one.

**Review by TheAlphamerc — CHANGES_REQUESTED** [review]
@aa-rodriguezv Thanks for your contribution. I have added some comments in some files please check those. 
One more thing that I want you ask you that in some please why you have change screen width to infinity ?

**Comment by aa-rodriguezv** [comment]
Hi there, will correct those. Width infinity, since it is not querying the device size, allows us to switch the wrapping widget to const, which is recommended wherever we can as a micro optimization.

File: android/app/src/debug/AndroidManifest.xml (hunk 2 above)
Comment (aa-rodriguezv) [reply android/app/src/debug/AndroidManifest.xml]: Fixed back to original

File: android/app/src/main/AndroidManifest.xml (hunk 3 above)
Comment (aa-rodriguezv) [reply android/app/src/main/AndroidManifest.xml]: Fixed back to original

**Comment by aa-rodriguezv** [comment]
Did the changes requested

**Review by TheAlphamerc — APPROVED** [review]

**Pull request merged**